js/init.js

```diff
@@ -161,7 +161,7 @@ define([
 		// window load event
 		// hide iOS browser chrome on load if hideUrlBar is true this is as fall back incase we were too early before
 		if ( $.mobile.hideUrlBar ) {
-			$window.load( $.mobile.silentScroll );
```

Comment:
Always use the long-form `.on()` calls. The event `.load()` method was deprecated a while back. 

https://jquery.com/upgrade-guide/3.0/#breaking-change-load-unload-and-error-removed